### PR TITLE
[NODE] feat: initial setup of bullMQ

### LINKED ISSUE #618: [node-pod] Add Queuing
# Description
(these requirements come from meeting with Dustin 12/7/22) We will have a POST endpoint which will accept something that will add the job to BullMQ with the package name. With BullMQ we will automatically listen for changes in the database and run the jobs in FIFO ordering. Jobs should run in worker threads.

Note: there is a possibility that Redis client does not work well with BullMQ, but this will be determined in further research. 

# Notes
General notes from specifications:

<meta charset="utf-8"><b style="font-weight:normal;" id="docs-internal-guid-41132257-7fff-aff7-9b4f-46315d612cd7"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size:11pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">As applications grow, the need for asynchronous workers become needed. Newer developers are less familiar with this pattern. As such, we want to include this as part of the starter kit as an opt-in feature so it can be easily used by developers.</span></p><br /><div dir="ltr" style="margin-left:0pt;" align="left">

Requirement | Description
-- | --
Select Queue Tech | There are several queue solutions on the market that serve different purposes. Select a queue that makes the most sense for the project and add it.
Infrastructure Setup | Regardless of the queue selected, a message queue will need to be utilized. For push queues, this is typically done with a formal message queue tech such as ElasticMQ, RabbitMQ, etc. For pull queues, this is typically done with Redis or equivalent in-memory storage. All of these require some level of local setup. Add this setup to the docker compose with setup instructions.
Add job generator endpoint | Add a POST /example-job endpoint that receives a message and adds it as a job to the queue to be processed.
Add worker/job processor | Add a worker that can process the job added to the queue. It should simply print the job message to STDOUT.
Add queue runner | Add or modify scripts to run the queue.

</div></b>

### PR DESCRIPTION
**Pull request by TapaiBalazs**
## Type of change

- [ ] Documentation change
- [ ] Bug fix
- [X] Feature

## Summary of change

- Added another redis instance
- set up bullMQ connection
- set up basic queue controller
- set up worker to run on different thread

## Checklist

- [x] These changes follow the [contributing guidelines](https://github.com/thisdot/starter.dev/blob/main/CONTRIBUTING.md)
- [x] This starter kit has been approved by the maintainers
- [x] Changes for this new starter kit are being pushed to an integration branch instead of main
- [x] All dependencies are version locked
- [x] This fix resolves #618 
- [x] I have verified the fix works and introduces no further errors

### CODE REVIEW

File: starters/express-typescript-typeorm-postgres/src/controllers/health/handlers/get-health.ts [hunk 1]
```diff
@@ -20,7 +22,15 @@ export async function getHealth(req: Request, res: Response): Promise<void> {
     return;
   }
 
-  if (redisPingResult.type === Result.ERROR) {
+  if (redisQueueHealthResult.type === Result.ERROR) {
+    res.status(StatusCodes.SERVICE_UNAVAILABLE).json({
```
Comment (dariodjuric) [inline starters/express-typescript-typeorm-postgres/src/controllers/health/handlers/get-health.ts:26]: Tip: you can just return this line and that way you don't need to have separate lines for setting the response and returning.

File: starters/express-typescript-typeorm-postgres/src/controllers/health/services/health.service.ts [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+        type: Result.ERROR,
+        message: error.message,
+        error,
+      };
+    });
+}
+
+function pingRedisQueueWithTimeout(): Promise<'PONG'> {
+  return defaultQueue.client.then((client: RedisClient) => {
+    return Promise.race([
```
Comment (dariodjuric) [inline starters/express-typescript-typeorm-postgres/src/controllers/health/services/health.service.ts:84]: Nice trick.

**Review by dariodjuric — APPROVED** [review]
LGTM

File: starters/express-typescript-typeorm-postgres/src/controllers/health/handlers/get-health.ts (hunk 1 above)
Comment (TapaiBalazs) [reply starters/express-typescript-typeorm-postgres/src/controllers/health/handlers/get-health.ts:26]: Yes, that's true, I'll do it in a separate PR though with all the handlers

**Pull request merged**